index.js

```diff
@@ -106,6 +106,43 @@ var Controllers = {
     };
   },
 
+  TabBarControllerIOS: function (id) {
+    return {
+      setHidden: function (params) {
+        return RCCManager.TabBarControllerIOS(id, "setTabBarHidden", params);
+      }
+    };
+  },
+
+  Modal: {
+    showLightBox: function(componentId, style) {
```

Comment:
Let's change the signature here to be similar to `Controllers.NavigationControllerIOS(id).push()`
So it's something like this:
`showLightBox: function(params)`
and you call it with:
`Modal.showLightBox({
  component: 'LightBoxExample',
  style: {}
});`

I found that this format is much better because it allows you and me to add more optional features in the future and maintain backwards compatibility in the signature.